projects/packages/sync/src/modules/class-themes.php

```diff
@@ -226,7 +224,8 @@ public function detect_theme_edit( $redirect_url ) {
 	 * @todo Refactor to use WP_Filesystem instead of fopen()/fclose().
 	 */
 	public function theme_edit_ajax() {
-		$args = wp_unslash( $_POST );
+		// This validation is based on wp_edit_theme_plugin_file().
+		$args = wp_unslash( $_POST ); // We need all of $_POST since we are validating all args, same as core does in wp_ajax_edit_theme_plugin_file.
```

Comment:
Same as the other one.

Reply:
Done via https://github.com/Automattic/jetpack/pull/37402/commits/7563c3fa74ad9b98aaf50e41fe19a237a1254f09